homeassistant/components/device_tracker/icloud.py

```diff
@@ -47,24 +47,22 @@ def keep_alive(now):
         api.authenticate()
         _LOGGER.info("Authenticate against iCloud")
 
-    track_utc_time_change(hass, keep_alive, second=0)
+    track_utc_time_change(hass, keep_alive, second=0, minute=4)
```

Comment:
Are you sure this one is firing and it is not a by product of the other callback ?